telegram/ext/_application.py

```diff
@@ -1431,12 +1432,15 @@ def add_handlers(
         .. versionadded:: 20.0
 
         Args:
-            handlers (List[:class:`telegram.ext.BaseHandler`] | \
-                Dict[int, List[:class:`telegram.ext.BaseHandler`]]): \
+            handlers (Sequence[:class:`telegram.ext.BaseHandler`] | \
+                Dict[int, List[:class:`telegram.ext.BaseHandler`] | \
+                Tuple[:class:`telegram.ext.BaseHandler`]]):
                 Specify a sequence of handlers *or* a dictionary where the keys are groups and
                 values are handlers.
-            group (:obj:`int`, optional): Specify which group the sequence of :paramref:`handlers`
-                should be added to. Defaults to ``0``.
```

Comment:
This should not be removed :)

Reply:
Oops, this was a mistake, not intentional 😅.